components/dot_menu/dot_menu.jsx

```diff
@@ -297,6 +297,7 @@ export default class DotMenu extends Component {
                         onClick={this.handlePinMenuItemActivated}
                     />
                     <MenuItemAction
+                        buttonId={`delete_button_post_${this.props.post.id}`}
```

Comment:
The ID here does not match with the one used at message_deletion_spec.js.  Also, we can use `id` like in [edit_post](https://github.com/mattermost/mattermost-webapp/pull/2412/files#diff-14335866591f7641b936717d36b9fae9R306),
```suggestion
                        id={`delete_post_${this.props.post.id}`}
```

Reply:
@letsila Thanks for the update. Last open item from me is above.  Please use `id` and revert unnecessary `buttonId` prop.

Reply:
@saturninoabril the `id` props on `<MenuItemAction>` is actually for the parent `<li>` element, that's why I needed to create `buttonId`.

Reply:
You're right and probably make sense but it has the same effect in doing with the parent <li> element.  It also makes consistent with how we're doing on [edit_post](https://github.com/mattermost/mattermost-webapp/pull/2412/files#diff-14335866591f7641b936717d36b9fae9R306).

Reply:
So you suggest that I remove `buttonId` which removes `id` on `<button>`? That would imply that I do `cy.get('#delete_post_').find('button').click()` in the e2e test spec.